spec/factories/user_factory.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :user do
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+    email { Faker::Internet.email }
+    password '123123123'
```

Comment:
This is ok, however, for you to know, it is possible to assign any string to password and then make password_confirmation equal to password